components/Chat.js

```diff
@@ -102,6 +92,7 @@ class Chat extends React.Component {
         // AsyncStorage.clear();
 
         // load convo history, if it exists
+        return;
```

Comment:
It seems the code below corresponding to correctly constructing chatbasemessages, @inealey or @OwenZhang001122 may want to check if it is fine to remove

Reply:
Also, %2 may not be a good way of calculation. A user may send multiple mesages while waiting for the response from chatbot